src/coord/src/client.rs

```diff
@@ -311,6 +311,36 @@ impl SessionClient {
             }
         }
 
+        fn datum_to_json(datum: &Datum, idx: usize, col_types: &[ColumnType]) -> serde_json::Value {
```

Comment:
Hmm, I feel there should be a TODO to relocate this method out of `src/coord` as it gets filled in more.

Reply:
We only recently moved this function into coord (https://github.com/MaterializeInc/materialize/commit/1a0b1353969d924a5638f1973dcc22325bfc1973). It is here because we wanted a single place to run this kind of stuff so that external callers (like the HTTP interface) would have a common way to do things.

Reply:
I see. Ok I'm now fine with this PR merging.